src/gui/src/PatternEditor/PianoRollEditor.cpp

```diff
@@ -491,50 +585,79 @@ void PianoRollEditor::addOrRemoveNote( int nColumn, int nRealColumn, int nLine,
 {
 	Note::Octave octave = (Note::Octave)nOctave;
 	Note::Key notekey = (Note::Key)nNotekey;
-	int nSelectedInstrumentnumber = Hydrogen::get_instance()->getSelectedInstrumentNumber();
-	Song *pSong = Hydrogen::get_instance()->getSong();
+	Hydrogen *pHydrogen = Hydrogen::get_instance();
+	int nSelectedInstrumentnumber = pHydrogen->getSelectedInstrumentNumber();
+	Song *pSong = pHydrogen->getSong();
 	Instrument *pSelectedInstrument = pSong->get_instrument_list()->get( nSelectedInstrumentnumber );
 
-	H2Core::Note* pDraggedNote = nullptr;
-	pDraggedNote = m_pPattern->find_note( nColumn, nRealColumn, pSelectedInstrument, notekey, octave );
-
-	int oldLength = -1;
-	float oldVelocity = 0.8f;
-	float oldPan_L = 0.5f;
-	float oldPan_R = 0.5f;
-	float oldLeadLag = 0.0f;
-	int oldNoteKeyVal = 0;
-	int oldOctaveKeyVal = 0;
+	Note* pFoundNote = pFoundNote = m_pPattern->find_note( nColumn, nRealColumn, pSelectedInstrument,
```

Comment:
Is that intentional?

Reply:
No, editing error.  I guess in coalescing a previously separate declaration and initialisation